frontend/widgets/OBSBasic_Transitions.cpp

```diff
@@ -1359,5 +1411,48 @@ int OBSBasic::GetOverrideTransitionDuration(OBSSource source)
 
 int OBSBasic::GetTransitionDuration()
 {
-	return ui->transitionDuration->value();
+	return transitionDuration;
+}
+
+void OBSBasic::SetCurrentTransition(const std::string &uuid, bool setTransition)
+{
+	auto transitionIter = transitions.find(uuid);
+
+	if (currentTransitionUuid == uuid || transitionIter == transitions.end())
+		return;
+
+	currentTransitionUuid = uuid;
+
+	if (setTransition)
+		SetTransition(transitionIter->second);
+
+	emit CurrentTranstionChanged(QString::fromStdString(uuid));
```

Comment:
Is the signal actually called `CurrentTranstionChanged` and not `CurrentTransitionChanged` (notice the missing `i`)?

Reply:
Uncaught typo, fixed